app/institutions/dashboard/-components/institutional-users-list/styles.scss

```diff
@@ -132,7 +132,7 @@
 .dropdown-panel {
     position: absolute;
     top: calc(100% + 5px);
-    right: 0;
+    right: 45%;
```

Comment:
This looks good in mobile view, but for desktop view, the dropdown shows up a little bit off-center to the left... I think you might be able to do something like `local-class='dropdown-panel {{is-mobile 'mobile'}}'` in the template, and then in here, you could add a rule like 
```
.dropdown-panel {
    // ..
    .mobile {
        right: 45%;
    }
}
```

Another possibility is we could use `ResponsiveDropdown` in the template instead. It would be a small refactor, but I think this may be the most robust fix. This way, we may be able to remove some of the position styling for this `.dropdown-panel` and let the `ResponsiveDropdown`'s default styling do the heavy lifting. In the template you could do:
```
<ResponsiveDropdown
    @buttonStyling={{true}}
    as |dd|
>
    <dd.trigger local-class='dropdown-trigger'>
        {{!-- FaIcon and translation here --}}
    </dd.trigger>
    <dd.content>
        {{!-- <div local-class='dropdown-panel'> and the rest here
       You'll have change `dd.actions.close` to just `dd.close`--}}
    </dd.content>
</ResponsiveDropdown>
    
```